src/itdelatrisu/opsu/ui/NotificationManager.java

```diff
@@ -50,7 +51,10 @@
 		public void click();
 	}
 
-	/** Notification. */
+	/** 
+	 * The default bubble notification,
```

Comment:
Not sure why you're changing these comments. Unnecessary "default" (no others) and they don't necessarily appear in the lower-right of the screen (if stacked).